### DISCUSSION TITLE
如何打开隐藏了的页面？

### DISCUSSION THREAD

**Discussion by huoyunbox** [Meloong-Git/PCL#1798] [提问] [answered]
我在玩的时候不小心隐藏了设置，我不知道怎么打开，想请教您解答。
![1](https://user-images.githubusercontent.com/132796458/236654934-e8654fc4-c9c2-4684-8e7b-3b5fa7fcbd33.png)

**Comment by MoYuan-CN** [comment, marked as answer]
按 F12
注:部分电脑是 Fn+F12

**Comment by huoyunbox** [comment]
谢谢。Get Outlook for AndroidFrom: ***@***.*** ***@***.***> on behalf of 凌云 ***@***.***>Sent: Sunday, May 7, 2023, 11:22 AMTo: Hex-Dragon/PCL2 ***@***.***>Cc: huoyunbox ***@***.***>; Author ***@***.***>Subject: Re: [Hex-Dragon/PCL2] 如何打开隐藏了的页面？ (Discussion #1798)
按 F12
